Source/Scene/Model.js

```diff
@@ -1724,6 +1756,23 @@ define([
         return shader;
     }
 
+    function createHighlightVertexShaderSource(vertexShaderSource) {
+        var renamedVS = ShaderSource.replaceMain(vertexShaderSource, 'czm_old_main');
+        // Modified from http://forum.unity3d.com/threads/toon-outline-but-with-diffuse-surface.24668/
+        var highlightMain = 'uniform float u_highlightSize;\n' +
+            'void main() \n' +
+            '{ \n' +
+            '  czm_old_main(); \n' +
+            '  vec3 n = normalize(v_normal);\n' +
```

Comment:
Is `v_normal` already normalized?

Is it safe to assume `v_normal` exists?  We likely need to get the name from the uniform semantic and then do the model-view transform here.